backend/app/services/knowledge/knowledge_permission.py

```
@@ -0,0 +1,141 @@
+    Returns:
+        True if the namespace has level='organization', False otherwise
+    """
+    # Special case: 'default' namespace is never organization-level
+    if namespace_name == "default":
+        return False
+
+    namespace = (
+        db.query(Namespace)
+        .filter(
+            Namespace.name == namespace_name,
+            Namespace.is_active == True,
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)